app/src/main/java/org/moire/opensudoku/gui/FolderListActivity.java

```diff
@@ -92,24 +94,30 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.folder_list);
-        View getMorePuzzles = findViewById(R.id.get_more_puzzles);
+        //View getMorePuzzles = findViewById(R.id.get_more_puzzles);
 
         setDefaultKeyMode(DEFAULT_KEYS_SHORTCUT);
         // Inform the list we provide context menus for items
         listView = findViewById(R.id.list_view);
         listView.setOnCreateContextMenuListener(this);
 
-        getMorePuzzles.setOnClickListener(v -> {
+        /*getMorePuzzles.setOnClickListener(v -> {
                 Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("http://opensudoku.moire.org"));
                 intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                 startActivity(intent);
-            });
+            });*/
 
         ValueView value_view = findViewById(R.id.value_view);
 
         ((OpenSudoku) this.getApplication()).setCurrentBalanceObserver(this, value_view);
         ((OpenSudoku) this.getApplication()).installTransactionObservers(this);
 
+        // TODO not sure where to put this at the moment
+        HTTPImportTask task = new HTTPImportTask(this);
+        task.execute();
+    }
```

Comment:
I imagine this will be in a separate class that controls and handles import games.